plotter/plot_datavalidation.py

```diff
@@ -148,20 +148,44 @@ def get_shape_sum(regions, process):
                 uncFile.ZG_PDF_met,
                 ]
             uncFile_photon = TFile(os.path.join(DIR,'../makeWorkspace/sys/photon_id_unc.root'))
+            uncFile_photon_scale = TFile(os.path.join(DIR,'../makeWorkspace/sys/photon_scale_unc.root'))
             if 'monojet' in category:
                 if year == 2017:
-                    uncertainties.append(uncFile_photon.monojet_2017_photon_id_extrap_up)
-                    uncertainties.append(uncFile_photon.monojet_2017_photon_id_up)
+                    h = uncFile_photon.monojet_2017_photon_id_extrap_up
+                    h.Scale(2)
```

Comment:
We probably need to do something more clever here. Scaling the histogram 2x is not equivalent to 2x the uncertainty variation, but leads to gigantic variations.